invenio.cfg

```diff
@@ -333,6 +334,8 @@ CDS_EOS_OFFLOAD_REDIRECT_BASE_PATH = ""
 CDS_CERN_SCIENTIFIC_COMMUNITY_ID = "CHANGE_ME"
 """The id of the CERN Scientific community."""
 
+CDS_CERN_MINT_ALTERNATE_IDS = {"cdsrn": "CDS Reference"}
```

Comment:
nitpick: please could you add a pydoc to explain what this does?

Reply:
added!